packages/vite-plugin-cloudflare/src/index.ts

```diff
@@ -1080,6 +1080,39 @@ if (import.meta.hot) {
 				}
 			},
 		},
+		// Plugin to handle cron/email/etc triggers
+		{
+			name: "vite-plugin-cloudflare:trigger-handlers",
+			async configureServer(viteDevServer) {
+				assertIsNotPreview(resolvedPluginConfig);
+
+				const entryWorkerConfig = getEntryWorkerConfig(resolvedPluginConfig);
+				assert(entryWorkerConfig, `No entry Worker config`);
+
+				const entryWorkerName = entryWorkerConfig.name;
+
+				viteDevServer.middlewares.use((req, res, next) => {
+					assert(req.url, "Request URL is undefined");
+					const url = new URL(req.url, UNKNOWN_HOST);
+
+					const requestHandler = createRequestHandler((request) => {
+						assert(miniflare, `Miniflare not defined`);
+
+						// email triggers
+						if (url.pathname === "/cdn-cgi/handler/email") {
+							// set the target service that handles these requests
+							// to point to the User Worker
+							// see `getTargetService` fn in packages/miniflare/src/workers/core/entry.worker.ts
+							request.headers.set(CoreHeaders.ROUTE_OVERRIDE, entryWorkerName);
+						}
```

Comment:
```suggestion
							// set the target service that handles these requests
							// to point to the User Worker
							// see `getTargetService` fn in packages/miniflare/src/workers/core/entry.worker.ts
							request.headers.set(CoreHeaders.ROUTE_OVERRIDE, entryWorkerName);
```

Don't think we need the condition here as this should always apply.